### ISSUE TITLE
White screen on authelia since 4.29.3

### ISSUE THREAD

**Issue by daften** [authelia/authelia#2020]
### Description

I have a docker-compose setup with authelia and traefik. This worked flawlessly so far. Since today, I can't use authelia anymore. Watchtower always upgrades automatically and it upgraded to 4.29.3 last night. If I revert to 4.29.2, things work perfectly again.

### Expected Behaviour

Authelia actually shows the UI

### Reproduction Steps

Upgrade to 4.29.3

### Additional Information

I see js not being executed in the browser console:
```
Refused to apply style from 'https://domain.name/static/css/2.c706b399.chunk.css' because its MIME type ('text/plain') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
Refused to apply style from 'https://domain.name/static/css/main.ecc43e7a.chunk.css' because its MIME type ('text/plain') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
Refused to execute script from 'https://domain.name/static/js/runtime-main.cc5273ce.js' because its MIME type ('text/plain') is not executable, and strict MIME type checking is enabled.
Refused to execute script from 'https://domain.name/static/js/2.22dd68b3.chunk.js' because its MIME type ('text/plain') is not executable, and strict MIME type checking is enabled.
Refused to execute script from 'https://domain.name/static/js/main.36b9809a.chunk.js' because its MIME type ('text/plain') is not executable, and strict MIME type checking is enabled.
```

The authelia logs show nothing out of the ordinary, debug logging is enabled.

**Comment by Dycelll** [comment]
I’ve been debuging this exact problem for the last hour. Can confirm the same problem. Traefik with authelia in the backend. Even exposing the port directly only results in a blank page.

**Comment by james-d-elliott** [comment]
I don't get this issue at all, are there any errors in the Authelia container logs?

**Comment by james-d-elliott** [comment]
So I tried with Chrome and had no issues. Can you confirm @Dycelll that you get the same console messages? Also what browser / version and proxy / version are you both using?

**Comment by james-d-elliott** [comment]
I have found part of the issue, I can probably manually revert changes from 4.29.3 until I figure out what fixes it. Strange that it affects you but not me however.

**Comment by daften** [comment]
I'm on the latest version of chrome on mac (90.0.4430.212)
Traefik is also the latest version within picodon (2.3.x)

**Comment by james-d-elliott** [comment]
Can you confirm these docker images fix it? They're not going to be the final fix, I reverted a package upgrade which I believe causes it. For some reason after bumping that package it's adding ***two*** content type headers, one correct one, and one 'text/plain' to all embedded files.

Docker Container

    docker pull authelia/authelia:revert-bump-fasthttp
    docker pull ghcr.io/authelia/authelia:revert-bump-fasthttp

**Comment by samip5** [comment]
This also occurs on Micirosoft Edge version 90.0.818.62. It's based on Chrome as well. I haven't yet tested the revert container tho.

**Comment by james-d-elliott** [comment]
Yeah it doesn't seem browser related, it's a bug with an upstream package. If someone can test the fix works I can look to put some time into getting it released.

**Comment by samip5** [comment]
> Can you confirm these docker images fix it? They're not going to be the final fix, I reverted a package upgrade which I believe causes it. For some reason after bumping that package it's adding _**two**_ content type headers, one correct one, and one 'text/plain' to all embedded files.
> 
> Docker Container
> 
> ```
> docker pull authelia/authelia:revert-bump-fasthttp
> docker pull ghcr.io/authelia/authelia:revert-bump-fasthttp
> ```

Seems to work with that container.

**Comment by james-d-elliott** [comment]
> 
> 
> > Can you confirm these docker images fix it? They're not going to be the final fix, I reverted a package upgrade which I believe causes it. For some reason after bumping that package it's adding _**two**_ content type headers, one correct one, and one 'text/plain' to all embedded files.
> > Docker Container
> > ```
> > docker pull authelia/authelia:revert-bump-fasthttp
> > docker pull ghcr.io/authelia/authelia:revert-bump-fasthttp
> > ```
> 
> Seems to work with that container.

Nice. Thanks for the feedback!

**Comment by Dycelll** [comment]
Sorry for the long response time, I was asleep. I just tested with ghcr.io/authelia/authelia:revert-bump-fasthttp and everything works as usual. If I can test or try anything, please let me know.

**Comment by kireque** [comment]
I can also confirm the authelia/authelia:revert-bump-fasthttp image fixes the problem

**Comment by james-d-elliott** [comment]
We have released 4.29.4 which should resolve it, I'm leaving this open until we get it properly fixed upstream. We just temporarily reverted the bump.

**Comment by Dycelll** [comment]
I can confirm the latest image also works. Thank you @james-d-elliott for your help and quick response!
